docs/Oracle Jobs/tasks.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: nodes.liquid
+date: Last Modified
+title: "Tasks"
+permalink: "docs/tasks/"
+whatsnext: {"Introduction to External Adapters":"/docs/external-adapters/", "Jobs":"/docs/jobs/"}
+---
+
```

Comment:
I think it's worth writing something short about how tasks replace the old adapters and where they fit from a conceptual standpoint in the new job format.